.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+packages.config                      @chocolatey-credits-reviewers
+CREDITS.md                           @chocolatey-credits-reviewers
+/src/chocolatey.resources/tools/     @chocolatey-credits-reviewers
+/lib/                                @chocolatey-credits-reviewers
```

This is referencing users, not teams.

These should probably be

```suggestion
packages.config                      @chocolatey/chocolatey-credits-reviewers
CREDITS.md                           @chocolatey/chocolatey-credits-reviewers
/src/chocolatey.resources/tools/     @chocolatey/chocolatey-credits-reviewers
/lib/                                @chocolatey/chocolatey-credits-reviewers
```

Also, the team needs to be created and have write permissions to this repository for it to work.

---

Ah, I didn't know it needed to be fully declared like that.  Yes, the group already exists, and I will verify permissions just now.